app/frontend/css/application.scss

```diff
@@ -113,37 +117,36 @@ $govuk-global-styles: true; // https://frontend.design-system.service.gov.uk/sas
     }
 
     a.defra-logo {
-      display: block;
+      display: table-cell;
       text-decoration: none;
       color: $govuk-text-colour;
-
-      @media (min-width: 48.0625em) {
-        display: table-cell;
-      }
     }
   }
 
   &-internal-service-name {
+    display: table-cell;
+    font-size: 20px;
```

Comment:
Know this was done before and not one to change now, but in general table-cell isn't really used anymore.  Flexbox works so well this is probably preferable.